src/System.IO.Abstractions.TestingHelpers/MockFileInfo.cs

```diff
@@ -137,7 +137,7 @@ public override DateTime LastAccessTimeUtc
             set
             {
                 var mockFileData = GetMockFileDataForWrite();
-                mockFileData.LastAccessTime = value;
+                mockFileData.LastAccessTime = value.ToLocalTime();
```

Comment:
Should we also do this for the other `*TimeUtc` properties?

Reply:
Not sure whether `ToLocalTime` is correct here. I think we rather want something like this: https://github.com/dotnet/runtime/blob/96d1c8e3ba4bc94afdcf15ee2896a3bc984820e7/src/libraries/System.Private.CoreLib/src/System/IO/FileSystemInfo.cs#L108

Reply:
@fgreinacher   Sorry for the late reply, just got back from vacation.

All other `*TimeUtc` properties already contain the `ToLocalTime` call. This way the behavior has been unified.

`File.File.GetUtcDateTimeOffset`cannot be used because it is internal. However, this behaves analogously to the proposed implementation, since it takes Utc for `DateTimeKind.Unspecified`.

Reply:
@fgreinacher  I've incorporated the recent changes from #875 and all test look fine.